config/initializers/assets.rb

```diff
@@ -17,8 +17,11 @@
   bootstrap-editable/clear.png
   application_body_api_docs.css
   intTelInput.css
+  libphonenumber/utils.js
   syntax_highlight.css
-  newflow_layout.css
+  newflow.css
   newflow_colors.css
   newflow.js
+  signup/email-value.coffee
+  newflow/signup_email_validations.coffee
```

Comment:
I believe this line is not necessary because this file is already being `//=require`d in the `newflow/index.js` file. Here: https://github.com/openstax/accounts/pull/786/files#diff-638c4a5c7e700ce5790610ac0dfec960R7